examples/ACI Unified Playground/frontend/src/components/ToolList.css

```
@@ -0,0 +1,113 @@
+  background-color: var(--primary);
+  color: var(--primary-foreground);
+  border-color: var(--primary);
+}
+
+.tool-list-error {
+  color: var(--destructive);
+  font-size: 0.9rem;
+  font-family: var(--font-sans);
+  padding: 0.75rem;
+  background-color: var(--destructive);
+  color: var(--destructive-foreground);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)